modules/apps/commerce/commerce-product-test/src/testIntegration/java/com/liferay/commerce/product/service/test/CPDefinitionLocalServiceTest.java

```diff
@@ -357,6 +362,50 @@ public void testAddCPDefinitionWithIgnoreSKUCombinationsAndDefaultInstance()
 		Assert.assertEquals(1, approvedCPInstances);
 	}
 
+	@Test
+	public void testAddExpiredCPDefinition() throws Exception {
```

Comment:
please add a test also for the updateStatus method, that was changed too..

Reply:
Added 3 more test that should cover the different cases.